server/src/main/java/org/cloudfoundry/identity/uaa/oauth/jwt/JwtClientAuthentication.java

```diff
@@ -74,33 +75,53 @@ public JwtClientAuthentication(KeyInfoService keyInfoService, OidcMetadataFetche
     this.oidcMetadataFetcher = oidcMetadataFetcher;
   }
 
-  public String getClientAssertion(OIDCIdentityProviderDefinition config) {
+  @VisibleForTesting
+  public String getClientAssertion(final OIDCIdentityProviderDefinition config) {
+    return getClientAssertion(config, false);
+  }
+
+  @VisibleForTesting
```

Comment:
make it public

Reply:
fixed with commit 68d3fdaf8389acf203480d331ad01fd93aa73e87